### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by dwendland**

### CODE REVIEW

File: src/test/java/org/keycloak/protocol/oid4vc/issuance/JAdESJwsSigningServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+                "Issuer DID should equal expected issuer");
+        assertTrue(jwtPayload.getOtherClaims().containsKey("vc"),
+                "Payload should contain VerifiableCredential vc.");
+
+        Map verifiableCredential = (Map) jwtPayload.getOtherClaims().get("vc");
+        assertEquals(signCredentialTestExpectedValues.vcIssuer(), verifiableCredential.get("issuer"),
+                "VC should contain issuer field with correct value");
+
+        // Verify signature
+        //Key publicKey = signingKey.getCertificateChain().get(0).getPublicKey();
```
Comment (pulledtim) [inline src/test/java/org/keycloak/protocol/oid4vc/issuance/JAdESJwsSigningServiceTest.java]: If this is covered by the verifySignature method, then why not delete this?

File: src/test/java/org/keycloak/protocol/oid4vc/issuance/JAdESJwsSigningServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+                kpg = KeyPairGenerator.getInstance("EC");
+                kpg.initialize(new ECGenParameterSpec(keyPairGenParameters.ecStdName()));
+                break;
+            default:
+                kpg = KeyPairGenerator.getInstance("RSA");
+                kpg.initialize(keyPairGenParameters.keySize());
+                break;
+        }
+
+        //String signerAlgorithm = "SHA256WithRSA";
```
Comment (pulledtim) [inline src/test/java/org/keycloak/protocol/oid4vc/issuance/JAdESJwsSigningServiceTest.java]: Remove pls if not needed anymore

File: src/test/java/org/keycloak/protocol/oid4vc/issuance/JAdESJwsSigningServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+        }*/
+        String signerAlgorithm = signatureAlgorithm.toString();
+
+        //KeyPairGenerator kpg = KeyPairGenerator.getInstance(keyGenAlgorithm);
+        //kpg.initialize(keySize);
+        var keyPair = kpg.generateKeyPair();
+
+        BigInteger serialNumber = BigInteger.valueOf(serial);
+        Instant validFrom = Instant.now();
+        Instant validUntil = validFrom.plus(10 * 360, ChronoUnit.DAYS);
```
Comment (pulledtim) [inline src/test/java/org/keycloak/protocol/oid4vc/issuance/JAdESJwsSigningServiceTest.java:348]: Not really important but something like "validFrom.plus(10, ChronoUnit.YEARS)" might be easier to read

**Review by pulledtim — APPROVED** [review]

File: src/test/java/org/keycloak/protocol/oid4vc/issuance/JAdESJwsSigningServiceTest.java (hunk 3 above)
Comment (dwendland) [reply src/test/java/org/keycloak/protocol/oid4vc/issuance/JAdESJwsSigningServiceTest.java:348]: This is throwing an exception: 
UnsupportedTemporalType Unsupported unit: Years

For a test probably not that important, therefore keeping it with DAYS.

**Review by wistefan — APPROVED** [review]

**Pull request merged**